### PR TITLE
Fix containerd checksum

### PR DESCRIPTION
**Pull request by tahsinrahman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
Fix containerd checksum
```

### CODE REVIEW

File: pkg/cloud/azure/services/config/containerdChecksum_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"k8s.io/klog"
+
```
Comment (justaugustus) [inline pkg/cloud/azure/services/config/containerdChecksum_test.go]: No space here.

File: pkg/cloud/azure/services/config/containerdChecksum_test.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/*
```
Comment (justaugustus) [inline pkg/cloud/azure/services/config/containerdChecksum_test.go]: Can we rename this file to `startupscript_test.go`?

File: pkg/cloud/azure/services/config/containerdChecksum_test.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	"os"
+	"os/exec"
+	"testing"
+
+	"k8s.io/klog"
+
+	"sigs.k8s.io/cluster-api/pkg/util"
+)
+
+func TestHash(t *testing.T) {
```
Comment (justaugustus) [inline pkg/cloud/azure/services/config/containerdChecksum_test.go]:
```suggestion
func TestContainerdSHA256(t *testing.T) {
```

**Comment by justaugustus** [comment]
Actually, let's address my nits in a future PR, as the SHA fix is critical.
/hold cancel
/priority critical-urgent
/milestone v1alpha1

**Comment by tahsinrahman** [comment]
@justaugustus should i do a seperate pr with changes according to your reviews?
￼

**Pull request merged**